client/maps/action-creators.ts

```diff
@@ -112,17 +116,24 @@ export function toggleFavoriteMap(
       () => {
         dispatch(
           openSnackbar({
-            message: map.isFavorited ? 'Removed from favorites' : 'Saved to favorites',
+            message: map.isFavorited
+              ? i18n.t('maps.server.favorites.removed', 'Removed from favorites')
+              : i18n.t('maps.server.favorites.saved', 'Saved to favorites'),
```

Comment:
we should make this "Added to favorites" (and change all the associated names + error)